### PR TITLE
Update policies.rst

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@KLuuKer  please review

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
Shouldn't that be 
```
#if NET451
        public static readonly Task CompletedTask = Task.FromResult(0);
#else
        public static readonly Task CompletedTask = Task.CompletedTask;
#endif
```

File: aspnet/security/authorization/policies.rst [hunk 1]
```diff
@@ -196,7 +196,7 @@ For example the previous ``BadgeEntryHandler`` could be rewritten as follows;
 Accessing MVC Request Context In Handlers
 -----------------------------------------
 
-The ``Handle`` method you must implement in an authorization handler has two parameters, an ``AuthorizationContext`` and the ``Requirement`` you are handling. Frameworks such as MVC or Jabbr are free to add any object to the ``Resource`` property on the ``AuthorizationContext`` to pass through extra information.
+The ``Handle`` method you must implement in an authization handler has two parameters, an ``AuthorizationContext`` and the ``Requirement`` you are handling. Frameworks such as MVC or Jabbr are free to add any object to the ``Resource`` property on the ``AuthorizationContext`` to pass through extra information.
```
Comment (blowdart) [inline aspnet/security/authorization/policies.rst]: Spelling.

**Comment by KLuuKer** [comment]
I'm don't work at the death star company but that aside

:shipit:

**Comment by Rick-Anderson** [comment]
@spboyer  please review.

**Pull request merged**